controller/userController.js

```diff
@@ -0,0 +1,120 @@
+const express = require("express");
+const bcrypt = require("bcrypt");
+const gravatar = require("gravatar");
+const jwt = require("jsonwebtoken");
+const { userSchema } = require("../validators/userValidator");
+const {
+  addNewUser,
+  findUserByEmail,
+  updateToken,
+  updateSub,
+  findUserById,
+} = require("../services/userService");
+const { SECRET_KEY } = process.env;
```

Comment:
```suggestion
const express = require("express"); // importy główne np fs

const bcrypt = require("bcrypt"); // importy bibliotek zewnętrznych które są npm install
const gravatar = require("gravatar");
const jwt = require("jsonwebtoken");

const { userSchema } = require("../validators/userValidator"); // importy własne
const { 
  addNewUser,
  findUserByEmail,
  updateToken,
  updateSub,
  findUserById,
} = require("../services/userService");
const { SECRET_KEY } = process.env;
```
+ sortowanie alfabetycznie. 
UPPER_CASE > camelCase > {someImport}

Reply:
To taki przykład. Zobacz komentarze. O to mi chodziło przy organizacji importów.